features/webdriver.yml

```diff
@@ -1,16 +1,19 @@
-draft_date: 2024-08-12
 name: WebDriver
-description: TODO
+description: WebDriver is a remote control interface that enables introspection and control of user agents by out-of-process programs.
 spec: https://w3c.github.io/webdriver/
+status:
+  compute_from: api.Navigator.webdriver
```

Comment:
Not reviewing the rest, but giving you heads up: I think this is very unlikely to be the right move. Literally no other part of this API is interoperable—just the part that tells you whether the browser is under the control of WebDriver. The "headline" view of WebDriver is not interoperable, so using a `compute_from` in this way is deeply misleading.

Reply:
Haha ok in that case I need dive deeper into how `compute_from` works, was just following the guide and what  Philip had showed me, figuring I'd grok it along the way. I'll read through the dist script.